Makefile

```diff
@@ -16,6 +16,11 @@
 # /bin/sh is POSIX compliant, ie it's not bash.  So let's be explicit:
 SHELL=/bin/bash
 
+INSTALL_DEV ?= true
```

Comment:
Nit: can we name this something like "INSTALL_DEV_REQS" and "INSTALL_TEST_REQS", for clarity?

Reply:
Fixed.